src/components/pages/novu-framework/code/code-with-tooltip/code-with-tooltip.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+
+import StepTooltip from './step-tooltip';
+
+const TOOLTIP_CONTENT = {
+  stepTooltip: <StepTooltip />,
+  echoTooltip: (
+    <span className="flex flex-col py-3.5">
+      <span className="whitespace-normal px-3.5">
+        (alias) <span className="text-yellow">new Echo</span>(&#123; apiKey, ... opts &#125;?:
```

Comment:
I see Echo here? Should this be Client instead?

..to match the new syntax/way of doing things.

Reply:
I'll change the code snippet once we have a new one from @scopsy. Just spoke w/ him about that this morning.